pom.xml

```diff
@@ -148,8 +148,10 @@
 
         <jetty9.version>${jetty94.version}</jetty9.version>
         <liquibase.version>4.25.1</liquibase.version>
-        <osgi.version>4.2.0</osgi.version>
-        <pax.web.version>7.1.0</pax.web.version>
+        <!-- matches quarkus 3.7.1 version and the pax.web.version -->
```

Comment:
Should we maybe add a comment here so we know why we're explicitly setting the version and not just relying on Quarkus?

Reply:
I'd maybe be rephrase a bit to be more explicit.
```suggestion
        <!-- matches quarkus 3.7.1 version and but also the pax.web.version, hence we can't rely on quarkus bom -->
```
Would this make sense?

Reply:
Sure. Also I didn't try to see if we could use later pax.web versions with an older osgi version, nor try to move ahead of the quarkus osgi version. This seemed like a good enough alignment for now.

Reply:
Thanks!